### ISSUE TITLE
@RequestBody not working when using more parameters

### ISSUE THREAD

**Issue by irbian** [stoicflame/enunciate#959]
This doesn´t work:

```
public ResponseEntity<InputStreamResource> dummy(@RequestBody MyType type, HttpServletRequest req, HttpServletResponse res) throws IOException {
```

This works:
```
public ResponseEntity<InputStreamResource> dummy(@RequestBody MyType type) throws IOException {
```

**Comment by irbian** [comment]
Looks like this doesn´t work either
```
    /**
     * @inputWrapped com.mycompany.MyType
     */
public ResponseEntity<InputStreamResource> dummy(@RequestBody MyType type, HttpServletRequest req, HttpServletResponse res) throws IOException {

```

**Comment by stoicflame** [comment]
I don't see any difference between the first two code snippets.

What does "work" mean? Like, show up in the docs?

Can you attach a diff to [enunciate-sample](https://github.com/stoicflame/enunciate-sample) that I can use to reproduce the issue?

**Comment by irbian** [comment]
You didn´t see any difference because I pasted twice the same code ^^U

Now it appears correctly

By "doesn´t works" I mean that the input type is not MyType but something strange and unrelated (something like status integer)

I´ll try to modify reproduce it in enunciate-sample

**Comment by stoicflame** [comment]
Fixed at aa704ee

**Comment by stoicflame** [comment]
[Enunciate 2.12](https://github.com/stoicflame/enunciate/releases/tag/v2.12.0) has been released.
